R/browse.R

```diff
@@ -59,6 +59,14 @@ browse_travis <- function(package = NULL, ext = c("org", "com")) {
   view_url(sub("github.com", travis_url, gh))
 }
 
+#' @export
+#' @rdname browse-this
```

Comment:
I think this would be clearer as `browse_circle_ci()` or `browse_circleci()`? (whichever punctuation you think is better)

Reply:
I don't know; Travis functions also do not have the "ci" suffix.
I don't have a strong preference on this though.

Reply:
We have an existing `use_circleci()` function FWIW.

Reply:
Just saw it, will therefore use the `circleci` pattern.